index.d.ts

```diff
@@ -2160,6 +2159,12 @@ declare namespace tracer {
      */
     maxContextOperations?: number,
 
+    /**
+     * Defines the pattern to ignore cookie names in the vulnerability hash calculation
+     * @default ".{32,}"
+     */
+    cookieFilterPattern?: boolean,
```

Comment:
I think this should be a string, not a boolean.

Reply:
OMG, true, also missed an update in `test.ts` to check the types